README.md

```diff
@@ -3,7 +3,19 @@
 Discover current Supervisor Services offered to support modern applications through vSphere Services.  
 New service will be added overtime with the goal to continue to empower your DevOps communities.
 
-*Supervisor Services are only available with Supervisor Clusters enabled using NSX-T*
+Supervisor Services **are only** available with Supervisor Clusters enabled **using NSX-T**.
+
+## Supported vSphere Versions
+
+| Supervisor Service | vSphere 7 | vSphere 8 |
+| :--: | :--: | :--: |
+| vSAN Data Persistence Platform Services - MinIO, Cloudian and Dell ObjectScale| ✅ | ✅ |
+| Backup \& Recovery Service - Velero | ✅ | ✅ |
+| Certificate Management Service - Cert-Manager | ✅ | ✅ |
+| Cloud Native Registry Service - Harbor | ❌ | ✅ |
```

Comment:
On 7.0 (and 8.0 until the deprecation) we supported harbor as the embedded harbor registry. Should we add a footnote here? I feel the user may think there is no way to deploy harbor on 7.0 if they only look at the table.

Reply:
Valid point 👍🏻 We could put an * in the cell and underneath the table the footnote.

Reply:
@YuxinZhou @AcidLeroy @jswolf059 

How do you like:

<img width="1031" alt="image" src="https://user-images.githubusercontent.com/31652019/225728598-a4589767-55b0-4be7-ae0a-4adc6e5b1408.png">